public/index.html

```diff
@@ -58,10 +58,10 @@ <h5 class="card-title">Business</h5>
                                 <div class="col-md-6" id="totalBusinessTask"></div>
                             </div>
                         </div>
-                    </div>
+                    </span>
```

Comment:
Please fix the span back to div this might create some problems with small screen